slsim/Util/agn_util.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+from astropy import constants as const
+
+
+def spin_to_isco(spin):
+    """Converts dimensionless spin parameter of a black hole to the innermost stable
+    circular orbit (in gravitational radii) [R_g = GM/c^2]
```

Comment:
What is the unit of R_g or dimensionless? It would be good to write the unit.

Reply:
Gotcha, R_g is units of length. In general for units I use astropy units (will not come into play until a mass is chosen anyway) so it may be expressed as anything equivalent to meters or centimeters for mks or cgs units.